### DISCUSSION TITLE
Vertical arranging textures in montagetextures op

### DISCUSSION THREAD

**Discussion by AdrianGreyish** [cables-gl/cables_docs#671] [Feature Requests] [answered]
Can the montageTextures operator receive a setting to arrange input textures vertically alongside the existing horizontal draw?

There are workarounds that involve rotating the textures, but a simple setting within the operator would be nice.

**Comment by pandrr** [comment, marked as answer]
there is now a new version on dev.cables.gl that can can arrange images in rows,columns or grid pattern and also simpler setting size of images

have fun

**Comment by AdrianGreyish** [comment]
Thank you for the addition, it will make things much easier in the future! I'll add it to my patch straight away :)
